### ISSUE TITLE
README.agda-lib expected

### ISSUE THREAD

**Issue by pnlph** [agda/agda-stdlib#827]
Hi,

i forked agda-stdlib and when trying to load (C-c C-l) files from the README folder, it seems that agda expects a .agda-lib file in the root.

How should i change my settings to contribute in README?

```
/Users/pnlph/.agda/libraries:2:
Failed to read library file
/Users/pnlph/Documents/GitHub/agda-stdlib/README.agda-lib
```

**Comment by gallais** [comment]
I typically locally modify the `standard-library.agda-lib` file like so:

```diff
< include: src
> include: src .
```

This lets me load and edit files in `README/`. I don't commit & push the
modification so that `README/` is still kept out of the library from the
users' point of view.

**Comment by MatthewDaggitt** [comment]
We should add this to the documentation!

**Comment by MatthewDaggitt** [comment]
Fixed with 2573f4e5
